src/ui/Windows/MainEdit.cpp

```diff
@@ -466,6 +466,22 @@ void DboxMain::OnDuplicateGroup()
         // Make copy normal to begin with - if it has dependents then when those
         // are added then its entry type will automatically be changed
         ci2.SetNormal();
+
+        // Set creation time to now but keep all others
+        // Note: potential date/time inconsistencies that should not be "fixed"
+        // during open validation i.e. fields changed before the entry was created!
+        ci2.SetCTime();
```

Comment:
Would be nice to have this section refactored into CItermData::SetDuplicateTimes(const CItemData &src);
for use in group and later in wx.